src/Symfony/Component/Validator/Resources/translations/validators.lt.xlf

```diff
@@ -432,15 +432,15 @@
             </trans-unit>
             <trans-unit id="111">
                 <source>The detected character encoding is invalid ({{ detected }}). Allowed encodings are {{ encodings }}.</source>
-                <target state="needs-review-translation">Nustatyta simbolių koduotė yra netinkama ({{ detected }}). Leidžiamos koduotės yra {{ encodings }}.</target>
+                <target>Aptikta simbolių koduotė yra netinkama ({{ detected }}). Leidžiamos koduotės yra {{ encodings }}.</target>
             </trans-unit>
             <trans-unit id="112">
                 <source>This value is not a valid MAC address.</source>
-                <target state="needs-review-translation">Ši vertė nėra galiojantis MAC adresas.</target>
+                <target>Ši reikšmė nėra tinkamas MAC adresas.</target>
             </trans-unit>
             <trans-unit id="113">
                 <source>This URL is missing a top-level domain.</source>
-                <target state="needs-review-translation">Šiam URL trūksta aukščiausio lygio domeno.</target>
+                <target>Šiam URL trūksta aukščiausio lygio domeno.</target>
```

Comment:
Maybe "Šioje nuorodoje trūksta aukščiausio lygio domeno." ?

Reply:
I don't think so, URL and "link" are not exactly the same thing. It's usually not translated in other languages, too.

Reply:
Yeah, but in lt URL could mean nothing for non-dev people :thinking: ok leave as is :)